### PR TITLE
update ie11 support info

### PR DESCRIPTION
**Pull request by jeffvg**
#### Changes proposed in this Pull Request:
- remove IE from core readme
- add updated verbiage to both angular and react pages browser support
- 

#### To Test:
-

### CODE REVIEW

File: src/docs/development/frameworks-web/angular.mdx [hunk 1]
```diff
@@ -126,12 +125,12 @@ PX Blue provides packages for code standards and style using ESLint and Prettier
 
 ## Browser Support
 
-Angular and Angular Material will work with any modern browser. For specific details, you can view their official support pages:
```
Comment (huayunh) [inline src/docs/development/frameworks-web/angular.mdx:129]: Supports Angular and Angular Material... not React.

**Review by huayunh — CHANGES_REQUESTED** [review]

File: src/docs/development/frameworks-web/angular.mdx (hunk 1 above)
Comment (jeffvg) [reply src/docs/development/frameworks-web/angular.mdx:129]: haha nothing like copy/paste

**Review by huayunh — APPROVED** [review]
Remember to run `yarn indexer`

**Review by joebochill — APPROVED** [review]

**Comment by joebochill** [comment]
> Remember to run `yarn indexer`

The indexer is going to get out of sync with all of the PRs going in to doc it right now anyway...we'll just need to make sure the last one in makes sure to do it.

**Pull request merged**